### PR TITLE
Change menu to reflect actual order of sections

### PR DESCRIPTION
**Pull request by gaeun**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Changed menu to reflect actual order of sections
- Live link: https://gaeun.github.io/gci16.fossasia.org/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,17 +42,16 @@
           <ul class="nav navbar-nav" id="menu">
             <li><a class="menu-item" href="#welcome">Welcome</a></li>
             <li><a class="menu-item" href="#map">Map</a></li>
+            <li><a class="menu-item" href="#stprojects">Projects</a></li>
+            <li><a class="menu-item" href="#loklak">Webtweets</a></li>
             <li><a class="menu-item" href="#mentors">Mentors</a></li>
-            <li><a class="menu-item" href="#contributors">Contributors</a></li>
```
Comment (robbyoconnor) [inline index.html:46]: Why did you remove this?

**Review by abishekvashok — APPROVED** [review]
Nice work guys @gaeun and @robbyoconnor

**Comment by abishekvashok** [comment]
You are quick! I was about to hit it
